apps/wallet-mobile/src/components/Warning/Warning.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+
+import {Icon} from '../Icon'
+import {Spacer} from '../Spacer'
+
+type Props = {content: string}
+
+export const Warning = ({content}: Props) => {
+  return (
+    <View style={styles.notice}>
+      <Icon.Info size={30} color="#ECBA09" />
```

Comment:
dont need the theme stuff here ? 🤔

Reply:
not sure if its feasible

Reply:
made the change now, was not making the change as the current colors were not in the theme but have changed them to match the ones we have now

Reply:
perfect !